src/Core/Tools/Services/Implementations/SendService.cs

```diff
@@ -308,6 +319,26 @@ private async Task ValidateUserCanSaveAsync(Guid? userId, Send send)
         }
     }
 
+    private async Task ValidateUserCanSaveAsync_vNext(Guid? userId, Send send)
+    {
+        if (!userId.HasValue)
```

Comment:
🤔 Should this branch also verify whether the user has any organizations, as happens on line 300? If not, why?

Reply:
I removed it because I thought it was an unnecessary leaking of the abstraction. The Send domain shouldn't need to know "if there are no org claims in the JWT then there are no policies and we don't have to check policy requirements". It should just get the policy requirement and let the Policy domain handle it (which it does).

It does make the method slightly less efficient, as checking the JWT could shortcut the db call, but it's not a particularly hot path.

All of that said - I don't feel strongly about it and can restore it if you disagree.

Reply:
I don't have a strong opinion either way. I think one of the things that lead me to wonder is that policies are based on orgs, but they're never used in the code. Since the DB can back into this information from a user, however, I think we're all good. :)